drivers/Timer.cpp

```diff
@@ -46,32 +46,39 @@ void Timer::stop() {
 
 int Timer::read_us() {
```

Comment:
That's an excellent question. I don't have the answer, I choose a conservative approach for this PR even if it hurts to this return type be int.